tests/main_tests/plot_test.py

```diff
@@ -0,0 +1,29 @@
+"""
+This script tests the plot function.
+"""
+import sys
+
+import matplotlib.pyplot as plt
+import pandas as pd
+
+sys.path.append("tests")
+sys.path.append("..src")
```

Comment:
Consider using `__init__.py` files where appropriate and possibly a `conftest.py` file to help remove the need for these `sys.path.append()`'s.

Reply:
removed these